.drone/lcov-report.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -e
+# set -x
+
+SKIPLIST="math"
+
+: "${LCOV_SKIP_PATTERN:='^[9]'}" # Set default lcov skip pattern
+
+pwd
+
+export CI_DIR=${BOOST_CI_SRC_FOLDER}/ci
+export LCOV_VERSION="v2.3"
+export LCOV_IGNORE_ERRORS_LEVEL=standard
+
+touch /tmp/failed.txt
+touch /tmp/succeeded.txt
+
+cd "$BOOST_ROOT"
+git submodule update --init --recursive --jobs 4
+./b2 headers
+
+# The script runcodecov.sh will be pieced together in parts, enabling variables
+# to be included into the contents of the script.
+
+# shellcheck disable=SC2016
+textpart1='#!/bin/bash
+set -x
```

Comment:
I think this won't be required once done as you have a lot of `echo`s in place. I prefer those instead of e.g. printing every echo effectively twice (through `-x`) making it hard to find the relevant output. Rather echo what failed if necessary.

Reply:
See latest changes.

`set -eu` added to lcov-report.sh.

It can't be included in runcodecov.sh because  
- `set -e` failures are expected, and caught
- `set -u` seems to interfere with enforce.sh:  

```
/drone/src/ci/enforce.sh: line 30: !new_varname: unbound variable
fatal: run_command returned non-zero status for libs/accumulators
```